README.md

```diff
@@ -27,9 +30,9 @@ Set of [semantic-release](https://github.com/semantic-release/semantic-release)
 
 ## Configuration
 
-Your credentials have to be configured with the environment variables `DOCKER_USERNAME` and `DOCKER_PASSWORD`.
+Your credentials have to be configured with the environment variables `DOCKER_USERNAME` and `DOCKER_PASSWORD`. If you are using a private docker registry, set its URL in the environment variable `DOCKER_REGISTRY`.
```

Comment:
This is not correct anymore, right?

Reply:
I think it still is. I looked at the beta api of semantic-release plugins and it seems unchanged. But a lof of things have been added to the beta release (channels etc.).

Reply:
I meant the DOCKER_REGISTRY env var, I don’t see that used anymore

Reply:
@felixfbecker Oops. You are correct. I forgot to take out the old change to the docs when I put in the new one. I just fixed that.